### PR TITLE
Support/add waifu token logo

### PR DESCRIPTION
**Pull request by demonslayer69420**
### 📝 Description
waifu2

Add logo for Waifu token
https://bscscan.com/token/0xd0aa796e2160ED260c668E90aC5f237b4eBd4b0d https://waifutoken.net/

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by demonslayer69420** [comment]
@Justkant kindly review the pr

File: .changeset/waifu-token.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"ledger-live-desktop": patch
+"@ledgerhq/live-common": patch
```
Comment (Justkant) [inline .changeset/waifu-token.md:3]: Please remove the `ledger-live-desktop` changeset as you only modified ledger-live-common
```suggestion
"@ledgerhq/live-common": patch
```

**Review by Justkant — CHANGES_REQUESTED** [review]
Small issue with the changeset, also I think you'll need to cleanup a bit the svg, we'll see what the CI tells us

**Comment by demonslayer69420** [comment]
@Justkant i have updated the change set, kindly review

**Comment by adrienlacombe-ledger** [comment]
@demonslayer69420 the filename must the same as the token ticker

**Comment by demonslayer69420** [comment]
> @demonslayer69420 the filename must the same as the token ticker

@adrienlacombe-ledger apologies, i have updated the file, sorry to keep troubling you over something this small

**Pull request closed without merging**